test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua

```diff
@@ -0,0 +1,49 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when WEB_VIEW AppHmiType is absent in application policies
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table not contain WEB_VIEW.
```

Comment:
@ydementieiev This expectation is redundant, please remove it.

Reply:
@aderiabin reworked [description](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/33588521b5f1ff45678b21bd871324afc6f42bd0)